target/linux/realtek/dts/rtl8382_hpe_1920-24g-poe-180w.dts

```diff
@@ -5,6 +5,18 @@
 / {
 	compatible = "hpe,1920-24g-poe-180w", "realtek,rtl838x-soc";
 	model = "HPE 1920-24G-PoE+ 180W (JG925A)";
+
+	gpio_fan_array {
+		compatible = "gpio-fan";
+		target-rpm = <10000>;
```

Comment:
This property is not listed in the [gpio-fan binding](https://elixir.bootlin.com/linux/v6.6.67/source/Documentation/devicetree/bindings/hwmon/gpio-fan.txt) for kernel v6.6.

It's only listed in the [fan-common binding](https://elixir.bootlin.com/linux/v6.9.12/source/Documentation/devicetree/bindings/hwmon/fan-common.yaml) since kernel v6.9, but the [updated gpio-fan binding](https://elixir.bootlin.com/linux/v6.9.12/source/Documentation/devicetree/bindings/hwmon/gpio-fan.yaml) does not include it. I also don't think a value of 10000 makes much sense, if that is neither of the available options.

I suggest to just drop it.